docs/DeveloperGuide.md

```diff
@@ -487,24 +503,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-*{More to be added}*
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-1.  Should be able to hold up to 35000 students (NUS undergraduate cohort size) and 35000 appointments without a noticeable sluggishness in performance for typical usage.
-1.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-1.  The product is not required to export statistics reports to PDF files.
-1.  There should be a mechanism to migrate data so that the application can be used with existing data on different machines.
+1.  **Accessibility**
+    * Vax@NUS should work on any _mainstream OS_ as long as it has Java `11` or above installed.
+1.  **Reliability**
+    * Vax@NUS should be able to hold up to 35000 students (NUS undergraduate cohort size) and 35000 appointments without a noticeable sluggishness in performance for typical usage.
+    * A user should be able to use Vax@NUS even while offline.
+1.  **Usability**
+    * A user with above average typing speed (41.4 words per minute) for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
```

Comment:
Should it be "most tasks" instead of "most of the tasks"; "a mouse" instead of "the mouse"?

Reply:
I think I will keep 'most of the tasks' because that's from AB3 but I will change it to 'a mouse' to match the line below it.